### PR TITLE
Nomis datasets added

### PR DESCRIPTION
**Pull request by dandwal**
### What

Nomis Metadata for datasets are now loadable and presentable on the website (note the user is directed to Nomis for the dataset contents).

It should look something like this:

Screenshot 2021-02-15 at 14 45 29

### How to review

First things first you will need Nomis datasets locally
You will need to navigate in terminal to your local `dp-dataset-api` ensure you have the latest version on the develop branch. Then run `make Nomis` this command will populate your local MongoDB instance with new datasets. These are stored in a similar way to CMD datasets.

You will then need to pull the following locally:
- https://github.com/ONSdigital/dp-frontend-dataset-controller/pull/204
- https://github.com/ONSdigital/dp-frontend-renderer/pull/566

You can now check that the Nomis datasets and CMD datasets are working correctly.
A Nomis dataset URL is the same as a CMD one like the following:
`/datasets/NM_144_1/editions/2011/versions/1`

To find a Nomis dataset name the easiest way is to open Robo 3T then under new connection navigate to datasets then collections double click datasets. Replace the 'NM_144_1' with a dataset name in this list, note you will see a mix of CMD and Nomis ones, the Nomis ones all start with 'NM'.

Note there are some issues with the Nomis dataset templates which are going to be resolved on the scraper side there are currently tickets open for these as a separate task. These are:
- Release date in the wrong format
- Usage notes not using correct markdown.
We are changing these on the scraper side rather than the frontend side so that our API users can also benefit from the fix.

There is also a version list page which contains just one entry (the current version of each edition of Nomis. This also has the same issue with the 'date' displaying wrong.

Another thing to note is that the Nomis template file is not reusing the filter landing page template. This is because the future design for the Nomis dataset page looks vastly different to the current CMD one, as such by putting in minor changes on every other line the template file would get into a mess. In addition, the CMD dataset pages are going to get a new design soon too. If they both end up looking similar in the end then we can merge them back together. But, for now, these are two separate dataset landing pages.

### Who can review

Anyone except me

### CODE REVIEW

**Review by cadmiumcat — COMMENTED** [review]
🕺  I can see the datasets are they look in your image, including the things you said were wrong e.g. the date formating 

All tests pass too. Is it worth adding tests for the `nomis` parts in `mapper.go`?

I'm happy with it, but I'm not very familiar with this or CMD so I'll suggest a second 👀

File: mapper/mapper.go [hunk 1]
```diff
@@ -69,6 +69,17 @@ func CreateFilterableLandingPage(ctx context.Context, req *http.Request, d datas
 	p.DatasetId = datasetID
 	p.ReleaseDate = ver.ReleaseDate
 	p.BetaBannerEnabled = true
+	if d.Type == "nomis" {
+		log.Event(ctx, "delete me", log.WARN, log.Data{"datasetDetails": d, "datasetVersion": ver})
```
Comment (sandypadmanabhan) [inline mapper/mapper.go:73]: If planning to keep the log , will it be better to have a different event name

Comment (dandwal) [reply mapper/mapper.go:73]: Haha, whoopsie

**Review by sandypadmanabhan — APPROVED** [review]
Looks good

**Pull request merged**